test/basic.c

```diff
@@ -102,3 +103,54 @@ TEST_CASE(rules_order) {
                              SYSCALL_ERRNO_SPEC(1));
   TEST_POLICY_BLOCKS_SYSCALL(SYSCALL_SPEC3(__NR_read, 5, 0, 1));
 }
+
+TEST_CASE(bitwise_operations) {
+  TEST_POLICY(
+      "#define O_WRONLY 1\n\
+       #define O_RDWR   2\n\
+       #define O_RDONLY 0\n\
+       #define O_CREAT  64\n\
+       POLICY a { \n\
+         ALLOW { exit, close, open { flags == O_WRONLY|O_CREAT }} \n\
+       } USE a DEFAULT KILL");
+  TEST_POLICY_ALLOWS_SYSCALL(SYSCALL_SPEC3(__NR_open, (long) "test_file", O_WRONLY|O_CREAT, 0600),
```

Comment:
this makes the test not fully isolated (e.g. there is "test_file" in cwd or cwd is not writable)
Use `__NR_getpid` and:
`getpid(arg) { arg == O_WRONLY|O_CREAT) }`
instead. Also no need to change harness and Makefile then.

Reply:
Switched to `read`. The test cases don't make much sense semantically since they operate on different parameters, but this is required to exercise code outside of the expression tree simplification step (which is why I missed the clobbered X-register issue).